.scalafmt.conf

```diff
@@ -0,0 +1,16 @@
+maxColumn = 80
+
+style = IntelliJ
```

Comment:
- using this option will require some changes to scalastyle-config.xml (due to the presence of additional spaces).